aten/src/THC/THCTensorScatterGather.cu

```diff
@@ -11,7 +11,7 @@
 // This version uses a static number of dimensions.
 template <typename IndexType, typename Real, int Dims>
 struct IndexToScatterGatherOffsets {
-  static __device__ void compute(
+  static inline __device__ void compute(
```

Comment:
do you know if any operation actually triggers this code? It looks like dead code to me, scatter/gather have been ported to ATen.